main.go

```diff
@@ -75,6 +78,10 @@ func main() {
 	flag.StringVar(&telegrafRequestsMemory, "telegraf-requests-memory", defaultRequestsMemory, "Default requests for memory")
 	flag.StringVar(&telegrafLimitsCPU, "telegraf-limits-cpu", defaultLimitsCPU, "Default limits for CPU")
 	flag.StringVar(&telegrafLimitsMemory, "telegraf-limits-memory", defaultLimitsMemory, "Default limits for memory")
+	flag.BoolVar(&enableIstioInjection, "enable-istio-injection", false,
+		"Enable injecting additional sidecar for monitoring istio sidecar container. If enabled, additional sidecar telegraf-istio will be added for pods with the Istio annotation enabled")
+	flag.StringVar(&istioOutputClass, "istio-output-class", "istio", "Class to use for adding telegraf-istio sidecar to monitor its sidecar")
+	flag.StringVar(&istioTelegrafImage, "istio-telegraf-image", "", "If specified, use a custom image for telegraf-istio sidecar")
```

Comment:
Maybe we can add the default here and remove the if here https://github.com/influxdata/telegraf-operator/pull/35/files#diff-6cc79099a212d91317d217d8088549beR421

Reply:
after discussion, let's table this